.bcr/presubmit.yml

```diff
@@ -23,6 +23,7 @@ tasks:
     - '-@apple_support//test:linking_opt_link_test'
     - '-@apple_support//test:binary_watchos_device_arm64e_test'
     - '-@apple_support//test:binary_watchos_device_arm64_test'
+    - '-@apple_support//test:binary_macos_binary_with_spaces_test'
```

Comment:
This wouldn't fix it, it's only failing in 7.x. I'm debugging.

Reply:
oh duh this list is for bazel 6, ill close this out